CreatubblesAPIClient/Sources/CreationUploadSession/CreationUploadSession.swift

```diff
@@ -318,6 +342,35 @@ class CreationUploadSession: NSObject, Cancelable
         currentRequest = requestSender.send(request, withResponseHandler: handler)
     }
     
+    private func uploadToGallery(error: ErrorType?, completion: (ErrorType?) -> Void)
+    {
+        if let error = error
+        {
+            completion(error)
+            return
+        }
+        if state.rawValue >= CreationUploadSessionState.SubmittedToGallery.rawValue
+        {
+            completion(nil)
+            return
+        }
+        
+        let request = GallerySubmissionRequest(galleryId: creationData.galleryId!, creationId: creation!.identifier)
```

Comment:
It's better to guard for a not set galleryId here instead of using yolo operator:
```swift
guard let galleryId = creationData.galleryId
else
{
state = .SubmittedToGallery
completion(nil)
return
}
...
```